apps/frontend/src/pages/not-found/not-found.tsx

```diff
@@ -9,9 +9,7 @@ const NotFound: React.FC = () => (
 		<p className={styles["text"]}>
 			The page you are looking for does not exist.
 		</p>
-		<Link to={AppRoute.ROOT} type="button">
-			Go to Homepage
-		</Link>
+		<Link to={AppRoute.ROOT}>Go to Homepage</Link>
```

Comment:
Why was this changed?

Reply:
my bad, it's maybe come when i try to merge with main branch